.travis.yml

```diff
@@ -2,11 +2,14 @@ language: python
 
 python:
   - 2.7
+  - 3.6
+  - 3.7
 
 env:
-  - TOXENV=django18
-  - TOXENV=django110
   - TOXENV=django111
+  - TOXENV=django20
+  - TOXENV=django21
+  - TOXENV=django22
```

Comment:
The failing test workers should be flagged as ok to fail via `allow_failures` until they can be fixed: https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail